Jenkinsfile

```diff
@@ -342,7 +342,6 @@ pipeline {
                     when {
                         beforeAgent true
                         allOf {
-                            branch 'main'
```

Comment:
@rbradford Looks like the VFIO worker is still not running with my change. May be it is a privilege thing that imposes on the PR owner instead of committer? Any advices to enable the VFIO worker temporarily here?

Reply:
Jenkins ignores changes to Jenkinsfile in PRs from users who don't have write access to the repository. Jenkins doesn't really know about separate commits and anyway the committer can be easily impersonated. If you want to test on the VFIO worker you would need to open you own PR.